legal-api/tests/unit/services/warnings/business/business_checks/test_firms.py

```diff
@@ -380,34 +379,34 @@ def test_check_completing_party(session, test_name, role, party_type, expected_c
         assert len(result) == 0
 
 
```

Comment:
I had to comment this out to avoid more changes because Paul is already working on these tests.